payment/src/main/java/org/killbill/billing/payment/core/PaymentGatewayProcessor.java

```diff
@@ -77,12 +77,15 @@ public PaymentGatewayProcessor(final OSGIServiceRegistration<PaymentPluginApi> p
     }
 
     public GatewayNotification processNotification(final String notification, final String pluginName, final Iterable<PluginProperty> properties, final CallContext callContext) throws PaymentApiException {
+
         return dispatchWithExceptionHandling(null,
+                                             pluginName,
                                              new Callable<PluginDispatcherReturnType<GatewayNotification>>() {
                                                  @Override
                                                  public PluginDispatcherReturnType<GatewayNotification> call() throws PaymentApiException {
-                                                     final PaymentPluginApi plugin = getPaymentPluginApi(pluginName);
+
                                                      try {
+                                                         final PaymentPluginApi plugin = getPaymentPluginApi(pluginName);
```

Comment:
Nit: Was there a reason to move that inside the try block (considering this throws a `PaymentApiException` and we catch `PaymentPluginApiException` ?